tests/savepoint/translate/translate_fv_update_phys.py

```diff
@@ -0,0 +1,365 @@
+import dataclasses
+from fv3core.testing import ParallelTranslateBaseSlicing
+import fv3core._config as spec
+import fv3core.utils.gt4py_utils as utils
+import fv3gfs.util as fv3util
+
+from fv3gfs.physics.stencils.fv_update_phys import ApplyPhysics2Dycore
+from fv3core.decorators import ArgSpec, FrozenStencil, get_namespace
+from fv3core.testing import ParallelTranslate2Py
+from fv3core.utils.typing import FloatField, FloatFieldIJ
+import numpy as np
+import copy
+
+
+@dataclasses.dataclass()
+class DycoreState:
+    u: FloatField
+    v: FloatField
+    delp: FloatField
+    ps: FloatFieldIJ
+    pe: FloatField
+    pt: FloatField
+    peln: FloatField
+    pk: FloatField
+    qvapor: FloatField
+    qliquid: FloatField
+    qice: FloatField
+    qrain: FloatField
+    qsnow: FloatField
+    qgraupel: FloatField
+    ua: FloatField
+    va: FloatField
+
+    def __getitem__(self, item):
+        return getattr(self, item)
+
+
+class TranslateFVUpdatePhys(ParallelTranslate2Py):
+    def __init__(self, grids):
+        super().__init__(grids)
+        grid = grids[0]
+        self._base.in_vars["data_vars"] = {
+            "u_dt": {},
+            "v_dt": {},
+            "t_dt": {},
+            "ua": {},
+            "va": {},
+            "u": grid.y3d_domain_dict(),
+            "v": grid.x3d_domain_dict(),
+            "qvapor": {},
+            "qliquid": {},
+            "qice": {},
+            "qrain": {},
+            "qsnow": {},
+            "qgraupel": {},
+            "peln": {
+                "istart": grid.is_,
+                "iend": grid.ie,
+                "jstart": grid.js,
+                "jend": grid.je,
+                "kend": grid.npz,
+                "kaxis": 1,
+            },
+            "delp": {},
+            "pt": {},
+            "ps": {},
+            "pe": {
+                "istart": grid.is_ - 1,
+                "iend": grid.ie + 1,
+                "jstart": grid.js - 1,
+                "jend": grid.je + 1,
+                "kend": grid.npz + 1,
+                "kaxis": 1,
+            },
+            "pk": grid.compute_buffer_k_dict(),
+            "edge_vect_e": {"dwind": True},
+            "edge_vect_n": {"dwind": True, "axis": 0},
+            "edge_vect_s": {"dwind": True, "axis": 0},
+            "edge_vect_w": {"dwind": True},
+            "vlat": {"dwind": True},
+            "vlon": {"dwind": True},
+            "es": {"dwind": True},
+            "ew": {"dwind": True},
+        }
+        self._base.out_vars = {
+            "qvapor": {},
+            "qliquid": {},
+            "qice": {},
+            "qrain": {},
+            "qsnow": {},
+            "qgraupel": {},
+            "pt": {},
+            "u": {},
+            "v": {},
+            "ua": {},
+            "va": {},
+        }
+
+    def collect_input_data(self, serializer, savepoint):
```

Comment:
All functions except for `compute_parallel` can be removed once Grid initialization is ready